packages/react-spinbutton/src/components/SpinButton/useSpinButtonStyles.ts

```diff
@@ -1,7 +1,17 @@
+import { SlotClassNames } from '@fluentui/react-utilities';
 import { makeStyles, mergeClasses } from '@griffel/react';
-import type { SpinButtonState } from './SpinButton.types';
+import type { SpinButtonSlots, SpinButtonState } from './SpinButton.types';
 
+/**
+ * @deprecated
+ */
 export const spinButtonClassName = 'fui-SpinButton';
```

Comment:
For `SpinButton` you can actually remove this class name instead of deprecating it since the component is not being exported from `@fluentui/react-components` yet:

```suggestion
```